packages/codemods/nuxt/4/template-compilation-changes/README.md

```diff
@@ -0,0 +1,42 @@
+## Template Compilation Changes
```

Comment:
instead of this heading, we can just have a 1 liner to explain what codemod does. [this 1 liner will show up in codemod registry... so users immediately understand what is it about]

This codemod removes lodash/template and related template utilities from Nuxt in favor of a more flexible and secure getContents() function for code generation in v3.

@35C4n0r

Reply:
on it.

Reply:
@alexbit-codemod made similar changes across all my open PRs